rclcpp/test/rclcpp/strategies/test_allocator_memory_strategy.cpp

```diff
@@ -816,16 +758,21 @@ TEST_F(TestAllocatorMemoryStrategy, get_next_timer_out_of_scope) {
 
 TEST_F(TestAllocatorMemoryStrategy, get_next_waitable_out_of_scope) {
   WeakNodeList nodes;
-  auto node = std::make_shared<rclcpp::Node>("node", "ns");
+  auto node = create_node_with_disabled_callback_groups("node");
   nodes.push_back(node->get_node_base_interface());
 
-  // Force waitable to go out of scope and cleanup after collecting entities.
   {
```

Comment:
Nit: now that we aren't using the scope to destroy the waitable, can we just get rid of this inner scope?

Reply:
I forget if the reset was required. I'm running a build to see if I can do without this line